cmd/ocget/main.go

```diff
@@ -0,0 +1,51 @@
+package main
```

Comment:
Personally I don't like the idea of shortening OnlineConf to OC in public interfaces and especially in the naming of command line tools. `ocget` name says nothing if placed alongside other binaries in `/usr/bin`.
I think `onlineconf-get` is better and in the line with `onlineconf-admin` and `onlineconf-updater`.